reference/functions/pgroonga-query-expand.md

```diff
@@ -129,6 +129,71 @@ SELECT pgroonga_query_expand('synonym_groups', 'synonyms', 'synonyms',
 -- (1 row)
 ```
 
+### Practical Example Using Synonym groups
+
+Sometimes you may want to search similar names like (Timothy or Tim), (William or Bill), (Stephen, Steven or Steve).
+
+Here are sample schema and data for solving this problem using Synonym groups.
+
+#### Name Table
+
+```sql
+CREATE TABLE names (
+  name varchar(255)
+);
+
+CREATE INDEX pgroonga_names_index
+          ON names
+       USING pgroonga (name pgroonga_varchar_full_text_search_ops_v2);
+
+INSERT INTO names
+  (name)
+  VALUES ('William Gates'),('Steven Paul Jobs'),('Timothy Donald Cook');
+```
+
+#### Synonym groups Table
+
+```sql
+CREATE TABLE synonym_groups (
+  synonyms text[]
+);
+
+CREATE INDEX synonym_groups_synonyms
+          ON synonym_groups
+       USING pgroonga (synonyms pgroonga_text_array_term_search_ops_v2);
+
+INSERT INTO synonym_groups
+  VALUES (ARRAY['Stephen', 'Steven', 'Steve']);
+```
+
+When you search "Steve" in this example, all of `"Stephen"`, `"Steven"` and `"Steve"` in query are expanded because PGroonga index is used.
```

Comment:
```suggestion
When you search "Steve" in this example, all of `"Stephen"`, `"Steven"` and `"Steve"` in the query are expanded because PGroonga index is used.
```